conda/environment-py38.yaml

```diff
@@ -13,3 +13,4 @@ dependencies:
   - requests
   - pytest
   - pytest-cov
+  - dask-geopandas
```

Comment:
Isn't it normal to only install this after the environment is built? I suppose maybe you want this to be sure to grab updated dependencies... but probably `pip install . --no-deps` is more thorough, no?

Reply:
i'm not sure. I added `dask-geopandas` here as the dependencies in *.yaml aren't sorted. Welcome to push to my branch or create an issue. Hoping to get agreement on https://github.com/raybellwaves/intake_geopandas/blob/to-dask/intake_geopandas/geopandas.py#L161-L214 in this PR.

Reply:
OK, forget it - I should have looked in the build log. It says that pip installation definitely ran after the conda stuff, although it shows up as [untagged](https://github.com/intake/intake_geopandas/runs/4191962825?check_suite_focus=true#step:4:11). 

I'm happy with the change itself, perhaps @ian-r-rose wants to have final say here.